### ISSUE TITLE
Migration fails because of duplicate detail fields in double_entry_lines

### ISSUE THREAD

**Issue by dersnek** [envato/double_entry#172]
## Steps to reproduce
Install gem with
```
gem "double_entry", github: "envato/double_entry"
```

Run
```
bundle install
```

Run
```
rails generate double_entry:install
```

Run
```
rails db:migrate
```

**Expected result:**
Migration should run successfully, adding required tables.

**Actual result:**
Migration fails with the following
```
== 20191106104924 CreateDoubleEntryTables: migrating ==========================
-- create_table("double_entry_account_balances", {:force=>true})
   -> 0.0392s
-- add_index("double_entry_account_balances", ["account"], {:name=>"index_account_balances_on_account"})
   -> 0.0176s
-- add_index("double_entry_account_balances", ["scope", "account"], {:name=>"index_account_balances_on_scope_and_account", :unique=>true})
   -> 0.0203s
-- create_table("double_entry_lines", {:force=>true})
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

you can't define an already defined column 'detail_id'.
...
.../db/migrate/20191106104924_create_double_entry_tables.rb:23:in `block in up'
...

Caused by:
ArgumentError: you can't define an already defined column 'detail_id'.
...
.../db/migrate/20191106104924_create_double_entry_tables.rb:23:in `block in up'
...
```

Exactly the same problem happens with detail_type field in the same table.

## Reason
On migration from integers to bigints, the following line was added:
```
t.references "detail", :index => false, :polymorphic => true
```

However, these lines weren't removed:
```
t.integer    "detail_id"
t.string     "detail_type"
```

So detail reference now looks like this in migration:
```
t.references "detail", :index => false, :polymorphic => true
t.integer    "detail_id"
t.string     "detail_type"
```

These are duplicate lines and they cause the problem.

_Added:_
If I remove these two lines, migration runs successfully.

**Comment by dersnek** [comment]
Guys? @orien? :)

**Comment by orien** [comment]
Fixed in #173! Thanks @dersnek
